core/node/bitswap.go

```diff
@@ -21,6 +22,7 @@ const (
 	DefaultTaskWorkerCount             = 8
 	DefaultEngineTaskWorkerCount       = 8
 	DefaultMaxOutstandingBytesPerPeer  = 1 << 20
+	DefaultProviderSearchDelay         = 20 * time.Millisecond
```

Comment:
Can you revert this to 1 second please ? Let try having this as an option before changing defaults please.